src/components/ClipBoardButton.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { FileCopy } from "@mui/icons-material";
+import Tooltip from "@mui/material/Tooltip";
+import IconButton from "@mui/material/IconButton";
+
+const ClipboardButton = ({ textToCopy }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const handleCopyToClipboard = (value) => {
+    navigator.clipboard.writeText(value);
+    setIsCopied(true);
+    setTimeout(() => {
```

Comment:
Do we want to store the timeout id returned as state and call clearTimeout before scheduling a new timeout to avoid odd behavior when a user might click it multiple times? If we don't do this, worst case is you might see it change from "Copied" to "Copy to clipboard" very quickly  from a previous timeout so not sure if it's worth the effort.